alibi/prototypes/tests/test_utils.py

```diff
@@ -0,0 +1,103 @@
+import pytest
+import numpy as np
+from alibi.utils.distance import squared_pairwise_distance, batch_compute_kernel_matrix
+from alibi.utils.kernel import GaussianRBF, GaussianRBFDistance, EuclideanDistance
+
+
+@pytest.mark.parametrize('size', [100])
+@pytest.mark.parametrize('ft_size', [2, 5, 10, 100])
+def test_squared_pairwise_distance(size, ft_size):
+    """
+    Test vectorize implementation of the squared pairwise distance
+    """
+    np.random.seed(0)
```

Comment:
Not sure we need to set the seed here and in other functions as it should work for any random array?

Reply:
It's not needed.